### PR TITLE
gRPC diagrams

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR adds the missing gRPC images reported in the #docs channel by @JPedro2. A quick fix to the release notes is also added.  The architecture page's structure has also been changed. Instead of a single tab group that displays the gRPC or the NATS architecture. Each section now contains its own tab to help the reader follow along easier.

## Review Changes

💻 [Preview URL](https://deploy-preview-1712--docs-spectrocloud.netlify.app/architecture/networking-ports)

### CODE REVIEW

File: docs/docs-content/architecture/networking-ports.md [hunk 1]
```diff
@@ -61,47 +92,34 @@ The following ports must be reachable from a network perspective for Palette to
...
-
-NATS is deprecated and will be removed in a future release. Starting with Palette 4.0.0, gRPC is used for all communication between the management platform and the workload cluster.
+<TabItem label="NATS" value="nats">
 
-:::
 
-## Management Platform
+### Management Platform
 
 |Port            |Direction|Purpose                   |
```
Comment (ritawatson) [inline docs/docs-content/architecture/networking-ports.md:105]:
```suggestion
|**Port**            |**Direction**|**Purpose**                   |    
```

File: docs/docs-content/architecture/networking-ports.md [hunk 2]
```diff
@@ -61,47 +92,34 @@ The following ports must be reachable from a network perspective for Palette to
...
+|HTTPS (tcp/443) |INBOUND        |Browser/API access to management platform|
+|NATS (tcp/4222) |INBOUND        |Agent running inside connecting to management platform|
 
 
-## Workload Cluster
+### Workload Cluster
 
 
-|Port |Direction | Purpose|
+|Port            |Direction | Purpose|
```
Comment (ritawatson) [inline docs/docs-content/architecture/networking-ports.md:114]:
```suggestion
|**Port**            |**Direction** | **Purpose**|
```

File: docs/docs-content/architecture/networking-ports.md [hunk 3]
```diff
@@ -110,60 +128,60 @@ You can expose inbound port 22 for SSH if you would like to access your cluster
...
 
+<Tabs groupId="architecture">
+<TabItem label="gRPC" value="gRPC">
 
-The following ports must be reachable from a network perspective for Palette to operate properly.
 
-## Management Platform
+### Management Platform
 
 |Port            |Direction|Purpose                   |
```
Comment (ritawatson) [inline docs/docs-content/architecture/networking-ports.md:159]:
```suggestion
|**Port**            |**Direction**|**Purpose**                   |    
```

File: docs/docs-content/architecture/networking-ports.md [hunk 4]
```diff
@@ -110,60 +128,60 @@ You can expose inbound port 22 for SSH if you would like to access your cluster
...
+|HTTPS (tcp/443) |OUTBOUND       |vSphere vCenter API,  Registry (packs, integrations), Pack containers, app updates, gRPC|
+|HTTPS (tcp/6443)|OUTBOUND       |Workload K8s cluster API Server|
 
 
-## Workload Cluster
+### Workload Cluster
 
 
-|Port            |Direction | Purpose|
+|Port |Direction | Purpose|
```
Comment (ritawatson) [inline docs/docs-content/architecture/networking-ports.md:170]:
```suggestion
|**Port** |**Direction** | **Purpose**|
```

File: docs/docs-content/architecture/networking-ports.md [hunk 5]
```diff
@@ -110,60 +128,60 @@ You can expose inbound port 22 for SSH if you would like to access your cluster
...
-The following ports must be reachable from a network perspective for Palette self-hosted to function correctly.
+<TabItem label="NATS" value="nats">
 
 
-![On-prem network diagram](/architecture_networking-ports_on_prem_network-diagram.png "#title="network diagram")
 
-## Management Platform
+### Management Platform
 
 |Port            |Direction|Purpose                   |
```
Comment (ritawatson) [inline docs/docs-content/architecture/networking-ports.md:186]:
```suggestion
|**Port**            |**Direction**|**Purpose**                   |    
```

File: docs/docs-content/architecture/networking-ports.md [hunk 6]
```diff
@@ -173,7 +191,7 @@ The following ports must be reachable from a network perspective for Palette sel
 |HTTPS (tcp/6443)|OUTBOUND       |Workload K8s cluster API Server|
 
 
-## Workload Cluster
+### Workload Cluster
 
 
 |Port |Direction | Purpose|
```
Comment (ritawatson) [inline docs/docs-content/architecture/networking-ports.md:197]:
```suggestion
|**Port** |**Direction** | **Purpose**|
```

File: docs/docs-content/architecture/networking-ports.md [hunk 7]
```diff
@@ -182,12 +200,22 @@ The following ports must be reachable from a network perspective for Palette sel
 |NATS (tcp/4222) |OUTBOUND       |Agent communication via message bus |
 |HTTPS (tcp/443) |OUTBOUND       |vSphere vCenter API, Registry (packs, integrations), Pack containers, Application updates.
```
Comment (ritawatson) [inline docs/docs-content/architecture/networking-ports.md:201]:
```suggestion
|HTTPS (tcp/443) |OUTBOUND       |vSphere vCenter API, Registry (packs, integrations), Pack containers, Application updates
```

Comment (ritawatson) [inline docs/docs-content/architecture/networking-ports.md:201]:
```suggestion
|HTTPS (tcp/443) |OUTBOUND       |vSphere vCenter API, Registry (packs, integrations), Pack containers, Application updates
```

File: docs/docs-content/release-notes.md [hunk 8]
```diff
@@ -13,7 +13,7 @@ tags: ["release-notes"]
 
 ## October 22, 2023 - Release 4.1.0
 
-Palette 4.1.0 introduces several new features and enhancements to the platform. New features, such as the automatic SSL control plane certificate renewal and node repavement warnings, are designed to continue to help you manage Kubernetes clusters with confidence. The new built-in notification system for our SaaS platform is another addition designed to help you stay up-to-date with announcements and changes by bringing the news directly to you. 
+Palette 4.1.0 introduces several new features and enhancements to the platform. New features, such as static IP address for Edge hosts, and node repavement warnings, are designed to continue to help you manage Kubernetes clusters with confidence. The new built-in notification system for our SaaS platform is another addition designed to help you stay up-to-date with announcements and changes by bringing the news directly to you.
```
Comment (ritawatson) [inline docs/docs-content/release-notes.md:16]:
```suggestion
Palette 4.1.0 introduces several new features and enhancements to the platform. New features, such as static IP address for Edge hosts and node repavement warnings, are designed to continue to help you manage Kubernetes clusters with confidence. The new built-in notification system for our SaaS platform is another addition designed to help you stay up-to-date with announcements and changes by bringing the news directly to you. 
```

**Review by ritawatson — APPROVED** [review]
Just some small fixes to address - table headers need boldface, and I saw the convention in these tables is no period. I removed a couple of remaining periods for consistency in this doc.

File: docs/docs-content/architecture/networking-ports.md [hunk 9]
```diff
@@ -110,60 +128,60 @@ You can expose inbound port 22 for SSH if you would like to access your cluster
...
 
-|Port            |Direction | Purpose|
+|Port |Direction | Purpose|
 |:---------------|:---------|:--------------|
 |HTTPS (tcp/443) |OUTBOUND | API access to management platform|
-|NATS (tcp/4222) |OUTBOUND       |Registry (packs, integrations), Pack containers, Application Updates|
-|NATS (tcp/4222) |OUTBOUND       |Registry (packs, integrations), Pack containers, Application Updates|
-
-:::info
+|NATS (tcp/4222) |OUTBOUND       |Agent communication via message bus. [Deprecated] |
```
Comment (caroldelwing) [inline docs/docs-content/architecture/networking-ports.md:173]:
```suggestion
|NATS (tcp/4222) |OUTBOUND       |Agent communication via message bus [Deprecated] |
```

I noticed that the other sentences don't have a final dot. Could we remove them to be consistent?

Comment (karl-cardenas-coding) [inline docs/docs-content/architecture/networking-ports.md:173]:
```suggestion
|NATS (tcp/4222) |OUTBOUND       |Agent communication via message bus [Deprecated] |
```

**Pull request merged**